packages/client/test/e2e.spec.ts

```diff
@@ -0,0 +1,141 @@
+/*
+  This test script (e2e.spec.ts) aims to perform integrated testing of the project. It executes a series of actions,
+  including deploying the contracts (registry, offchain resolver, and universal resolver), creating the Client using Viem, 
+  and initializing the gateway locally. After deploying and configuring the contracts, the Client can access
+  off-chain information during the tests. It's important to note that this initial test script only sets up the
+  environment and stops at the gateway call. It still requires implementing the connection between the gateway and 
+  layer two, or the gateway and the database.
+*/
+
+// Importing abi and bytecode from contracts folder
+import {
+  abi as abiOffchainResolver,
+  bytecode as bytecodeOffchainResolver,
+} from '../../contracts/out/OffchainResolver.sol/OffchainResolver.json'
+import {
+  abi as abiRegistry,
+  bytecode as bytecodeRegistry,
+} from '../../contracts/out/ENSRegistry.sol/ENSRegistry.json'
+import {
+  abi as abiUniversalResolver,
+  bytecode as bytecodeUniversalResolver,
+} from '../../contracts/out/UniversalResolver.sol/UniversalResolver.json'
+
+import { ethers as eth } from 'hardhat'
+import { BaseContract, Contract } from 'ethers'
+// import { beforeAll, it, describe, expect } from 'vitest'
+import { normalize, labelhash, namehash } from 'viem/ens'
+import { localhost } from 'viem/chains'
+import { createTestClient, http, publicActions, BaseError } from 'viem'
+import { HardhatEthersSigner } from '@nomicfoundation/hardhat-ethers/signers'
+import { expect } from 'chai'
```

Comment:
given that `describe`, `before`, and `it` aren't being imported, should this be?

Reply:
yes. The expect is not default. Needed to be imported.